src/sentry/static/sentry/app/views/settings/project/filtersAndSampling/filtersAndSampling.tsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import {RouteComponentProps} from 'react-router';
+import isEqual from 'lodash/isEqual';
+
+import ExternalLink from 'app/components/links/externalLink';
+import {t, tct} from 'app/locale';
+import {Organization, Project} from 'app/types';
+import {DynamicSamplingRules, DynamicSamplingRuleType} from 'app/types/dynamicSampling';
+import AsyncView from 'app/views/asyncView';
+import SettingsPageHeader from 'app/views/settings/components/settingsPageHeader';
+import TextBlock from 'app/views/settings/components/text/textBlock';
+import PermissionAlert from 'app/views/settings/organization/permissionAlert';
+
+import TransactionRules from './transactionRules';
+import {getPlatformDocLink} from './utils';
+
+type Props = RouteComponentProps<{projectId: string; orgId: string}, {}> &
+  AsyncView['props'] & {
+    organization: Organization;
+  };
+
+type State = AsyncView['state'] & {
+  project: Project | null;
+  transactionRules: DynamicSamplingRules;
+};
+
+class FiltersAndSampling extends AsyncView<Props, State> {
+  getTitle() {
+    return t('Filters & Sampling');
+  }
+
+  getDefaultState() {
+    return {
+      ...super.getDefaultState(),
+      transactionRules: [],
+      project: null,
+    };
+  }
+
+  getEndpoints(): ReturnType<AsyncView['getEndpoints']> {
+    const {orgId, projectId} = this.props.params;
+    return [['project', `/projects/${orgId}/${projectId}/`]];
```

Comment:
You should be able to get the project from the ProjectContext. You could either use the legacy context API, or use`withProject()`

Reply:
The idea was to modify this endpoint soon to get the project Details, but you are right... I will replace it with the `withProject() ` for now 😉